zip-0320.rst

```diff
@@ -5,7 +5,242 @@
   Owners: Daira Emma Hopwood <daira@electriccoin.co>
           Kris Nuttycombe <kris@nutty.land>
   Credits: Hanh
-  Status: Reserved
+  Status: Draft
   Category: Standards / Wallet
   Discussions-To: <https://github.com/zcash/zips/issues/757>
 
+Terminology
+===========
+
+The key words "MUST", "MUST NOT", "SHOULD", and "MAY" in this document are
+to be interpreted as described in BCP 14 [#BCP14]_ when, and only when, they
+appear in all capitals.
+
+The terms below are to be interpreted as follows:
+
+Abstract
+========
+
+This ZIP defines a new encoding for transparent Zcash addresses. Wallets must
+ensure that no shielded notes are spent in transactions that send to a
+transparent address encoded in the specified fashion.
```

Comment:
Asking the awkward questions: if the requirement is to be able to return the funds, why do we need to require that no shielded notes are spent, as opposed to just ensuring that there is at least one transparent input?

Reply:
Binance has repeatedly stressed that they do not want to receive any funds directly from the shielded pool.